src/java.desktop/share/legal/freetype.md

```diff
@@ -533,46 +554,34 @@ consider it more useful to permit linking proprietary applications with the
 library.  If this is what you want to do, use the GNU Library General
 Public License instead of this License.
 
-#########################################################################
-
---- fthash.c and fthash.h are covered by the following notices ---
-
-/*
- * Copyright 2000 Computing Research Labs, New Mexico State University
- * Copyright 2001-2015
```

Comment:
Why did this notice need to be moved down to the end of the file ?
It makes the diff look bigger than it actually is and I can't imagine any reason we would need to change the order.

Reply:
This is due to the addition of the MIT license being treated as a separate license from the freetype license. In the modified tool, licenses are separated by license type. The additional freetype licenses seems more logically related to freetype licenses so it follows the freetype & GPL license. Then, other licenses (such as MIT in this case) follow afterwards.

This can be manually modified to maintain the old order but I believed it made more sense this way in the tool since variants of the primary license would follow the primary license.

Reply:
Well if it is only a one-off...

BTW phrases such as "the modified tool" etc are very out of place here .. I don't see a tool in this PR, do you ?
I believe you are referring to some script(s) that not part of OpenJDK that you are using to produce these updates.